ResearchKit/Charts/ORKGraphChartView.h

```diff
@@ -106,20 +109,6 @@ ORK_AVAILABLE_DECL
 - (NSInteger)graphChartView:(ORKGraphChartView *)graphChartView numberOfPointsForPlotIndex:(NSInteger)plotIndex;
```

Comment:
@rsanchezsaez Yeah, you are right. Sure, let's use `numberOfDataPointsForPlotIndex:`;
And are you going to rename `ORKRangedPoint` -> `ORKValueRangeDataPoint`
and
`ORKValueStack` -> `ORKValueStackDataPoint`

Maybe you can convince me we don't need `DataPoint` in the value names:)